virttest/utils_misc.py

```diff
@@ -536,10 +536,13 @@ def get_path(base_path, user_path):
     :param base_path: The base path of relative user specified paths.
     :param user_path: The user specified path.
     """
-    if os.path.isabs(user_path) or aurl.is_url(user_path):
+    if aurl.is_url(user_path):
         return user_path
```

Comment:
seems that the result will be affected, if the 2 lines are deleted.
such as "http://www.baidu.com" is the parameter, os.path.isabs("http://www.baidu.com") wil be False, then following if sentence will got errors.
if not os.path.isabs(user_path):
        user_path = os.path.join(base_path, user_path)

if i am wrong, please correct me ,thanks.